indico/modules/users/client/js/Favorites.jsx

```diff
@@ -0,0 +1,171 @@
+                        position="bottom center"
+                      />
+                    </div>
+                  </List.Item>
+                ))}
+              </List>
+            ) : (
+              <div styleName="empty-favorites">
+                <Translate>You have not marked any user as favourite.</Translate>
+              </div>
+            )}
+          </div>
+        </div>
+        <PersonLinkSearch
+          existing={Object.values(favoriteUsers).map(u => u.identifier)}
+          onAddItems={e => e.forEach(u => addFavoriteUser(u.userId))}
+          disabled={false}
+          withExternalUsers={false}
+          withFavorites={false}
+          triggerFactory={searchTrigger}
+        />
+      </div>
+      <div className="column col-50">
+        <div className="i-box just-group-list">
+          <div className="i-box-header">
+            <div className="i-box-title">
+              <Translate>Favourite Categories</Translate>
+            </div>
+          </div>
+          <div className="i-box-content">
+            {favoriteCats !== null && Object.keys(favoriteCats).length > 0 ? (
+              <List relaxed celled styleName="fav-list">
+                {Object.values(favoriteCats).map(cat => (
+                  <List.Item key={cat.id} styleName="fav-item">
+                    <List.Content>
+                      <div styleName="list-flex">
+                        <div>
+                          <a href={cat.url} target="_blank" rel="noopener noreferrer">
+                            {cat.title}
+                          </a>
+                          <br />
+                          <TooltipIfTruncated>
+                            <span styleName="category-path">
+                              {cat.chain_titles.slice(0, -1).join(' » ')}
+                            </span>
+                          </TooltipIfTruncated>
+                        </div>
+                        <Popup
+                          trigger={
+                            <Icon
+                              styleName="delete-category"
+                              name="close"
+                              onClick={() => deleteFavoriteCat(cat.id)}
+                              link
+                            />
+                          }
+                          content={Translate.string('Remove from favourites')}
+                          position="bottom center"
+                        />
+                      </div>
+                    </List.Content>
+                  </List.Item>
+                ))}
+              </List>
+            ) : (
+              <div styleName="empty-favorites">
+                <Translate>You have not marked any category as favourite.</Translate>
+              </div>
+            )}
+          </div>
+        </div>
+      </div>
+    </>
+  );
+}
+
+FavoriteCatManager.propTypes = {
+  userId: PropTypes.number.isRequired,
```

Comment:
not required & default to null (due to the change i suggested in the template in order to only pass a user if if it's not the current user like we do on the profile picture page)